functions/controllers/onboardingController.js

```diff
@@ -0,0 +1,221 @@
+const functions = require("firebase-functions");
+const admin = require("firebase-admin");
+const {logger} = functions;
+const DEBUG = process.env.DEBUG;
+
```

Comment:
This is not a Firebase function, but an internal function, which means it will not be exported nor used by the frontend. That is the reason I decided not to document it. Documentation is shared externally (front end).

Reply:
Sure, thanks for pointing that out.